hackathon_site/dashboard/frontend/src/components/orders/OrderCard/OrderCard.tsx

```diff
@@ -13,7 +13,10 @@ const OrderCard = ({ teamCode, orderQuantity, time, id }: OrderProps) => {
     const date = new Date(time);
     const month = date.toLocaleString("default", { month: "short" });
     const day = date.getDate();
-    const hoursAndMinutes = date.getHours() + ":" + date.getMinutes();
+    const hours = date.getHours() + ":";
```

Comment:
I don't think you're using these constants anywhere, so you can remove them

Reply:
Which constants? I'm using it just below